packages/myst-parser/src/tokensToMyst.ts

```diff
@@ -9,6 +9,22 @@ import { u } from 'unist-builder';
 import { MarkdownParseState, withoutTrailingNewline } from './fromMarkdown.js';
 import type { MdastOptions, TokenHandlerSpec } from './types.js';
 
+export function computeAmsmathTightness(
```

Comment:
I think this is very similar to the other "compute tightness" function, but it's a little hard to be certain since the function is so complex. Can we just reuse that function and have the `+1` shift (with comment) in the ams math handler...?

Reply:
I started off that way, but opted to duplicate. If we change the underlying library for amsmath, this is the only place that should be impacted. I think this is appropriately guarded by tests, so am going to opt to leave as is!